package.json

```diff
@@ -43,83 +44,66 @@
   "dependencies": {
     "array-equal": "^1.0.0",
     "bezier-easing": "^2.0.3",
-    "chroma-js": "^1.1.1",
     "classnames": "^2.2.3",
     "color": "^0.11.3",
     "d3-interpolate": "^1.0.0",
-    "date-arithmetic": "^3.1.0",
-    "lodash.assign": "^4.0.9",
     "lodash.camelcase": "^4.1.0",
```

Comment:
I've been doing some testing with the lodash webpack and babel plugins. It has both pros and cons. 

Pros: 
Neater package.json (less deps), less import statements for components that uses multiple lodash functions, -3kB bundle size

Cons: 
When altering the set of used lodash functions we have to manually maintain the lodash-webpack-plugin options object by removing or adding features if needed.

I'm updating the PR